cbmc/proofs/IotMqtt_DeserializeSuback/Makefile

```diff
@@ -28,11 +28,12 @@ DEF += -DCBMC=1 -DCBMC_PROOF_DESERIALIZE_SUBACK=1
 ################################################################
 # Loop unwinding
 
+LOOP += _decodeSubackStatus.0:$(BUFFER_SIZE)
+LOOP += IotListDouble_FindFirstMatch.0:$(SUBSCRIPTION_COUNT_MAX)
+LOOP += IotListDouble_RemoveAllMatches\$$link1.0:$(SUBSCRIPTION_COUNT_MAX)
```

Comment:
This is adding loop unwindings?

Reply:
Yes, there is a path that calls `IotListDouble_RemoveAllMatches`, so we need to add these loop unwindings, otherwise, the proof will fail (see `_decodeSubackStatus` function).